File: 프로그래머스/1/12918. 문자열 다루기 기본/문자열 다루기 기본.java
class Solution {
    public boolean solution(String s) {
        boolean answer = true;
        char tmp;
		
		if(s.length() != 4 && s.length() != 6) {
			answer = false;
            // s의 길이가 4 혹은 6이 아닐시 false를 리턴하고
		}
		
		for(int i = 0; i < s.length(); i++) {
			tmp = s.charAt(i);
		    // char 변수인 tmp에 s를 넣어 한글자씩 판별하고
            
			if(Character.isDigit(tmp) == false) {
				answer = false;
			} // tmp가 숫자가 아니라면 false를 리턴한다
		}
        return answer;
    }
}

File: 프로그래머스/1/12918. 문자열 다루기 기본/문자열 다루기 기본.js
function solution(s) {
    var answer = true;
    
    if (s.length !== 4 && s.length !== 6) {
        answer = false;
        // s의 길이가 4 또는 6이 아니면 false를 반환합니다.
    }
    
    for (let i = 0; i < s.length; i++) {
        let tmp = s.charAt(i);
        // s의 각 문자를 tmp에 저장합니다.
        
        if (!/\d/.test(tmp)) {
            answer = false;
        } // tmp가 숫자가 아니라면 false를 반환합니다.
    }
    return answer;
}
